cluster/addons/cloud-controller-manager/cloud-node-controller-role.yaml

```diff
@@ -80,6 +80,12 @@ rules:
   verbs:
   - list
   - watch
+- apiGroups:
+    - ""
```

Comment:
The indents looks a bit off. Might need to check your editor settings.

Reply:
Updated